src/lib/core/ripple/ripple.ts

```diff
@@ -100,7 +100,7 @@ export class MatRipple implements OnChanges, OnDestroy {
   }
 
   ngOnChanges(changes: SimpleChanges) {
-    if ((changes['trigger'] || changes['_matRippleTrigger']) && this.trigger) {
+    if (changes['trigger'] && this.trigger) {
```

Comment:
You should be able to switch this to `change.trigger` as well.

Reply:
Done. Great idea